web/modules/custom/joinup_workflow/joinup_workflow.module

```diff
@@ -246,3 +266,26 @@ function joinup_workflow_is_motivation_required($entity_type_id, $workflow_id, $
 
   return !empty($required_transitions[$entity_type_id][$workflow_id]) && in_array($transition_id, $required_transitions[$entity_type_id][$workflow_id]);
 }
+
+/**
+ * Change notification when saving content as draft.
+ *
+ * @param array $form
+ *   The form array.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   The form state object.
+ */
+function joinup_workflow_message_form_submit_alter(array &$form, FormStateInterface $form_state) {
+  $node = $form_state->getFormObject()->getEntity();
+  if ($node->get('field_state')->getString() == 'draft') {
+    \Drupal::messenger()->deleteByType('status');
+    $args = [
+      '@type' => node_get_type_label($node),
+      '%title' => $node->getTitle(),
+      '%state' => $node->get('field_state')->getString(),
+      ':user' => Url::fromRoute('user.page')->toString(),
+      ':url' => $node->toUrl('canonical')->toString(),
+    ];
+    \Drupal::messenger()->addStatus(t('@type <a href=":url">%title</a> has been created as %state. You can find it in the section "My unpublished content" located in your <a href=":user">My account page</a>, or in the aforementioned section under the Collection/Solution it was created in.', $args));
```

Comment:
I think the request intent was to have either _Collection_ or _Solution_, but not _Collection/Solution_. I think, but I am not sure, that you can get that with:

```
$node->get('oe_audience')->entity->get('rid')->entity->label()
```